services/node_manager/src/main/java/com/futurewei/alcor/nodemanager/service/NodeService.java

```diff
@@ -0,0 +1,20 @@
+package com.futurewei.alcor.nodemanager.service;
+
+import com.futurewei.alcor.nodemanager.entity.NodeInfo;
+
+import java.util.Hashtable;
+import java.util.List;
+
+public interface NodeService {
+    int getNodeInfoFromFile(String path) throws Exception;
```

Comment:
As we discuss, we plan to support bulk add through APIs, instead of a local file path. The file path (relative or absolute) is one parameter in the REST call which essentially will send the entire file (in a serialized way) to NodeManager, then NodeManager diseralizes it back to a recognized format (e.g. NodeInfoJson).

Reply:
This module is added.
Test: curl -v -F file=@/mnt/d/dev/alcor/config/machine.json http://localhost:8080/nodes/upload
"d/dev/alcor/config/machine.json" : file name example

Reply:
This is a good change. Thank you!